main.rb

```diff
@@ -0,0 +1,25 @@
+require './app'
+
+def print_question
+  puts "Please choose an option by enterin a number:
+    1- List all books.
+    2- List all people.
+    3- Create a person.
+    4- Create a book.
+    5- Create a rental.
+    6- List all rentals for a given person id.
+    7- Exit."
+end
+
+class Main
+  app = App.new
+  puts 'Welcome to library Application!'
+  loop do
+    print_question
+    selected_option = app.select_opt
+    break if selected_option == 7
+  end
+  puts 'Thanks for using this App!!'
+end
+
+Main
```

Comment:
- [ ] Kindly run your app and ensure it works before submitting it for review. None of these options works at all. From options 1 to 6. Only exiting the app, option 7, works.

  I cannot create people, books or rentals. I cannot list any of them either. None works at all. See example of error message.

  ![image](https://github.com/obibaadoma/School_Library/assets/54848363/86cd354b-562a-482d-abd7-9e5b9133d585)